notebooks/pluvial/Hydro4_samples.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
Here are my bulleted comments for this notebook:

- When choosing the 10%, 10%, 90%, and 90% deciles for the quartiles, why do you say this is conservative? I would think being conservative would be the most extreme case which is the 10%.
- Update Inputs description.
- Can delete hydromet_conv import and some of the other paths to clean up the notebook.
- No need to fill select_distro na with zero, there should never be an na value
- No need to convert a dataframes columns to a list before enumerating, and you can just pass the variable directly to the string instead of using the index of the list of columns--that is the power of enumerating.

Reply:
Renamed to limiting case (versus conservative case). Select_distro na deleted. Enumerate has been fixed.  hydromet_conv import has been deleted